satellite/marketingweb/server_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package marketingweb_test
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/sync/errgroup"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+)
+
+type CreateRequest struct {
+	Path   string
+	Values url.Values
+}
+
+func TestCreateOffer(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+
+		requests := []CreateRequest{
+			{
+				Path: "/create/referral-offer",
+				Values: url.Values{
+					"Name":                      {"Referral Credit"},
+					"Description":               {"desc"},
+					"ExpiresAt":                 {"2119-06-27"},
+					"InviteeCreditInCents":      {"50"},
+					"InviteeCreditDurationDays": {"50"},
+					"AwardCreditInCents":        {"50"},
+					"AwardCreditDurationDays":   {"50"},
+					"RedeemableCap":             {"150"},
+				},
+			}, {
+				Path: "/create/free-credit-offer",
+				Values: url.Values{
```

Comment:
We need to add a test case that the user will have input like `InviteeCreditInCents: "50.5"`